generate-requirements-full.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+pip install -r requirements-core.txt
```

Comment:
`pip-compile` is going to use the `requirements-core.txt` as provided in its own temporary / internal virtual environment.

```suggestion
```